### PR TITLE
longer list ellipsize

### PR DESCRIPTION
**Pull request by marcellourbani**
Tomasz,
this is the longer ID change made with ellipsize.
I left a smaller font (24 vs 32 I reckon) as with the bigger one was showing only 2 digits when shortening.

Marcello

### CODE REVIEW

File: app/src/main/res/layout/issue_list_elem.xml [hunk 1]
```diff
@@ -32,13 +32,13 @@
         android:layout_height="wrap_content"
         android:layout_alignParentLeft="true"
         android:layout_below="@+id/project_shortcut"
-        android:ellipsize="none"
+        android:ellipsize="start"
+        android:scrollHorizontally="true"
```
Comment (tmszdmsk) [inline app/src/main/res/layout/issue_list_elem.xml]: Sorry Marcello for such a delay, I took holidays and there was an easter also.

It looks ok, but I don't know what this does
```
android:scrollHorizontally="true"
``` 
It doesn't change anything (at least on my phone) even if I have very long issue id (like 6 digits).

**Comment by marcellourbani** [comment]
No worries,I took my time too.
Looks like it only means something if ellipsize="marquee", but that would hide the last digits rather than the first.
I will remove that line when I have a minute.

Marcello

File: app/src/main/res/layout/issue_list_elem.xml (hunk 1 above)
Comment (marcellourbani) [inline app/src/main/res/layout/issue_list_elem.xml]: Removed

**Comment by marcellourbani** [comment]
Done

**Pull request merged**